app/components/H5/H5.tsx

```diff
@@ -13,10 +13,7 @@ export const H5: FunctionComponent<TProps> = ({
   className: propsClassName,
   ...props
 }) => {
-  const componentClassName = cn(
-    "sm:text-4xl text-[26px] font-unboundedMedium",
-    propsClassName,
-  );
+  const componentClassName = cn("sm:text-4xl text-[26px]", propsClassName);
```

Comment:
```suggestion
  const componentClassName = cn("sm:text-4xl text-2xl", propsClassName);
```
should be the `text-2xl`, please [check the page](https://tailwindcss.com/docs/font-size), also you can check the doc of how you can setup autocomplete for your [editor](https://tailwindcss.com/docs/editor-setup)

<img width="799" alt="image" src="https://github.com/user-attachments/assets/f8f89517-065c-44d6-9e37-044f7a4553e6">